devel/libjwt/Portfile

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+name                libjwt
+version             1.8.0
+revision            0
```

Comment:
We do not put `revision 0` in portfiles. If the revision is 0, just leave the line out.